synchronizer/synchronizer_test.go

```diff
@@ -668,68 +700,72 @@ func expectedCallsForsyncTrustedState(t *testing.T, m *mocks, sync *ClientSynchr
 		Once()
 
 	m.State.
-		On("BeginStateTransaction", sync.ctx).
+		On("BeginStateTransaction", mock.Anything).
```

Comment:
I don't fully understand this change

Reply:
Do you mind the change of `sync.ctx` to `mock.Anything`? Is because I think that It's not important which ctx are you sending, and forcing to be this one could make that it breaks in the future if we decide to use another one.